bluebottle/bb_projects/static/js/bluebottle/bb_projects/routes.js

```diff
@@ -104,8 +103,8 @@ App.MyProjectListRoute = Em.Route.extend(App.ScrollToTop, App.TrackRouteActivate
 App.MyProjectRoute = Em.Route.extend({
     // Load the Project
     model: function(params) {
-        var store = this.get('store');
-        if (params.id == 'new' || params.id == 'null') {
+        var store = this.get('store'); 
+        if (params.id == 'new' || params.id == 'null' || params.id == 'undefined') {
```

Comment:
Do you know how this can become 'undefined'? It will look pretty ugly in the url.